src/ApiService/ApiService/onefuzzlib/NotificationOperations.cs

```diff
@@ -21,7 +21,7 @@ public class NotificationOperations : Orm<Notification>, INotificationOperations
     private IEvents _events;
 
     public NotificationOperations(ILogTracer log, IStorage storage, IReports reports, ITaskOperations taskOperations, IContainers containers, IQueue queue, IEvents events)
-        : base(storage)
+        : base(storage, log)
     {
         _log = log;
```

Comment:
Since `_logTracer` is `protected` on the base, couldn't we remove this line and use `_logTracer` instead?

Reply:
i accidentally merged before pushing the code change.
will be addressed in the next PR